assets/css/style.css

```diff
@@ -43,4 +63,27 @@
 .brand .name {
     font-size: 24px;
 }
-/* End of Brand */
```

Comment:
I think we should have .branh .icon here event if we did not need it at the moment